src/gui/generalsettings.cpp

```diff
@@ -294,18 +295,46 @@ void GeneralSettings::slotUpdateInfo()
 
 void GeneralSettings::slotUpdateChannelChanged(const QString &channel)
 {
-    ConfigFile().setUpdateChannel(channel);
-    auto *updater = qobject_cast<OCUpdater *>(Updater::instance());
-    if (updater) {
-        updater->setUpdateUrl(Updater::updateUrl());
-        updater->checkForUpdate();
-    }
+    if (channel == ConfigFile().updateChannel())
+        return;
+
+    auto msgBox = new QMessageBox(
+        QMessageBox::Warning,
+        tr("Change update channel?"),
+        tr("The update channel determines which client updates will be offered "
+           "for installation. The \"stable\" channel contains only upgrades that "
+           "are considered reliable, while the versions in the \"beta\" channel "
+           "may contain newer features and bugfixes, but have not yet been tested "
+           "thoroughly."
+           "\n\n"
```

Comment:
Does the wording here matter? Since we don't do betas.. Stable would be the new features release and beta would be the release candidates?

Reply:
Yes indeed. I didn't dare change it.